pkg/provisioner/templates/nv-driver.go

```diff
@@ -34,21 +34,27 @@ wget https://developer.download.nvidia.com/compute/cuda/repos/$distribution/x86_
 sudo dpkg -i cuda-keyring_1.1-1_all.deb
 
 with_retry 3 10s sudo apt-get update
-install_packages_with_retry cuda-drivers
+install_packages_with_retry cuda-drivers{{if .Version}}={{.Version}}{{end}}
 
 nvidia-smi -L
```

Comment:
A nit here, but we should probably drop the `-L` so that we can also see the driver version being applied.